libcob/ChangeLog

```diff
@@ -3606,6 +3638,14 @@
 	  I need to come up with a better way to synchronize separate
 	  processes so I can reliably recreate the test cases
 
+2016-12-24  Ron Norman <rjn@inglenet.com>
+
+	* fileio.c [WITH_DB]: reordered struct indexed_file, adding file_lock_set
+	  and use it to fix multiple unlocking
+	* fileio.c [WITH_DB]: extracted bdb_open_cursor, bdb_close_cursor,
+	  bdb_close_index, lock_file
+	* fileio.c [WITH_DB]: added code for BDB < 4.6
+
```

Comment:
these entries up and again the test note "At present..." removed completely

Reply:
Done.